charts/posthog/values.yaml

```diff
@@ -415,10 +415,10 @@ kafka:
     # -- Enable persistence using PVC
     enabled: true
     # -- PVC Storage Request for kafka volume
-    size: 5Gi
+    size: 20Gi
   # -- A size-based retention policy for logs
-  # -- Should be less than kafka.persistence.size, ideally 70-80%
-  logRetentionBytes: _4_000_000_000
+  # -- Should be less than kafka.persistence.size, ideally ~50%
+  logRetentionBytes: _10_000_000_000
   # -- The minimum age of a log file to be eligible for deletion due to age
   logRetentionHours: 24
```

Comment:
Should we change this as well? Didn't we recently change some topic settings to be in the order of weeks? How does this cluster setting plays with that? Is it a global override or a default for topics without an explicit retention policy?

Reply:
:+1: I would definitely bump this up to a week or so. The `logRetentionBytes` will be the constraining factor for larger customers and we should communicate that in the docs that if they want to have a buffer for downtime, increase the [PVC claim size](https://github.com/PostHog/charts-clickhouse/pull/145/files#diff-e5d75a211cc4472486a4b35840f9e60d261b574e167ab44decf6bf5e4b7c3606R418)

Reply:
It's the minimum retention before allowing someone to be deleted, which means that if we set this to 1 week, then anything that's only 6 days old can't be deleted even if the disk is completely full

Reply:
me too & everyone else 😅  Related comment thread with more info https://github.com/PostHog/vpc/pull/110#discussion_r642531233

Reply:
see https://github.com/PostHog/charts-clickhouse/issues/166